cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewRequest.java

```diff
@@ -19,15 +19,20 @@
 import io.cdap.cdap.proto.artifact.AppRequest;
 import io.cdap.cdap.proto.id.ProgramId;
 
+import javax.annotation.Nullable;
+
 /**
  * Represents the preview application request.
- * @param <T> the type of application config
  */
-public class PreviewRequest<T> {
+public class PreviewRequest {
```

Comment:
Can you explain a bit why make PreviewRequest non-parameterized will help the preview perf?

Reply:
It doesn't. But it is never get used. So it is useless to be parameterized.